R/bundlePackageRenv.R

```diff
@@ -0,0 +1,142 @@
+snapshotRenvDependencies <- function(bundleDir,
+                                     extraPackages = character(),
+                                     verbose = FALSE) {
+  recordExtraDependencies(bundleDir, extraPackages)
+
+  old <- options(renv.verbose = FALSE, pkgType = "source")
+  defer(options(old))
+
+  renv::snapshot(bundleDir, prompt = FALSE)
+  defer(removeRenv(bundleDir))
+
+  parseRenvDependencies(bundleDir, snapshot = TRUE)
+}
+
+parseRenvDependencies <- function(bundleDir, snapshot = FALSE) {
+  renv <- jsonlite::read_json(renvLockFile(bundleDir))
```

Comment:
I wonder if I should expose something like `renv::lockfile$read()` so you can read an `renv` lockfile and let `renv` handle parsing of the Repositories field

Reply:
https://github.com/rstudio/renv/issues/1220